### PR TITLE
Create basic example app

### LINKED ISSUE #6: Create basic example app

### PR DESCRIPTION
**Pull request by DanielPotor**
closes #6

### CODE REVIEW

File: .expo/README.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+> Why do I have a folder named ".expo" in my project?
```
Comment (lgnstn) [inline .expo/README.md:1]: Can you please add instructions for how you can run this example project?

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1 @@
+node_modules
```
Comment (lgnstn) [inline .gitignore:1]: Please  also add `.DS_Store` to `.gitignore`.

File: example/Avoidable.js [hunk 3]
```diff
@@ -0,0 +1,201 @@
+import React from 'react';
```
Comment (lgnstn) [inline example/Avoidable.js:1]: This file will be removed after the lib is deployed to npm, right?

**Review by lgnstn — CHANGES_REQUESTED** [review]

File: .expo/README.md (hunk 1 above)
Comment (DanielPotor) [reply .expo/README.md:1]: I will create a separate issue for this.

File: example/Avoidable.js (hunk 3 above)
Comment (DanielPotor) [reply example/Avoidable.js:1]: Yes. This file is copied over until we can use it from npm

**Review by lgnstn — APPROVED** [review]

**Pull request merged**